Build/Remotion.Local.build

```diff
@@ -36,24 +41,26 @@
     <SkipNuGet>True</SkipNuGet>
     <SkipDependDB>True</SkipDependDB>
   </PropertyGroup>
-  
+
```

Comment:
Revert whitespace change?

Reply:
No, because they are unneccessary. Boy scout rule, you know ;-)

Reply:
MK always tells us to revert unnecessary whitespace changes, that's why I'm asking

Reply:
Yes, probably if the white spaces are added, but in this case a mistake is corrected.

Reply:
No, if our IDE corrects a mistake from the past we usually revert it because it is irrelevant to the change set of the PR, but here it's so small that it probably doesn't matter anyway

Reply:
Nevertheless, it's a mistake, and it's only one change. I wouldn't correct all such mistakes in the change set, though, because this would be a task of its own.

Reply:
Both is correct. New whitespace-errors must be fixed whenever they're noticed during the review. Existing mistakes should not be fixed when the spot is unrelated to the other changes happening. 

The occasional random fix in a file that has numerous other changes is okay and a matter of preference for the task owner and reviewer.  In the end it comes down to whether the change introduces noise to the change set and hides the significant code changes. That's why fixes in otherwise unchanged files are a definite no-no, same goes for changes that are a significant percentage of the file's total changes. 

So, in this instance, yes, I don't care, there's  enough 'necesary' changes in the file that this individual fix doesn't hurt the readability of the changeset.

Reply:
Oh, and I do reserve the right to change my mind in similar situations based on the amount of mental stimulants present in the bloodstream during the review. AKA it comes down to a gut feeling if I want it reverted. Sorry! but I probably won't argue that it needs to be reverted when challenged.